# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1296940164 28800
#      Sat Feb 05 13:09:24 2011 -0800
# Branch release-3-4-x
# Node ID aafbaea219cb3b43c122c9eed2038c13aca9acb4
# Parent  2427541a8408ba36400891797b7cc80ed825e43c
Update list of functions not requiring a docstring.

diff --git a/doc/ChangeLog b/doc/ChangeLog
--- a/doc/ChangeLog
+++ b/doc/ChangeLog
@@ -1,8 +1,13 @@
+2010-02-05  Rik  <octave@nomad.inbox5.com>
+
+	* interpreter/doccheck/mk_undocumented_list: Add dbnext, alias for
+	dbstep, to functions not requiring a docstring.
+
 2010-02-05  Rik  <octave@nomad.inbox5.com>
 
 	* interpreter/doccheck/aspell-octave.en.pws: Add new spelling
 	exceptions to Octave private dictionary.
 
 2010-02-05  Rik  <octave@nomad.inbox5.com>
 
 	* interpreter/octave.texi: Update copyright on manual to 2011.
diff --git a/doc/interpreter/doccheck/mk_undocumented_list b/doc/interpreter/doccheck/mk_undocumented_list
--- a/doc/interpreter/doccheck/mk_undocumented_list
+++ b/doc/interpreter/doccheck/mk_undocumented_list
@@ -76,16 +76,17 @@ bessel
 besselh
 besseli
 besselk
 bessely
 bug_report
 chdir
 comma
 debug
+dbnext
 exit
 F_DUPFD
 F_GETFD
 F_GETFL
 finite
 fntests
 F_SETFD
 F_SETFL
